aiven/provider.go

```diff
@@ -28,6 +28,7 @@ func Provider() terraform.ResourceProvider {
 			"aiven_database":                     datasourceDatabase(),
 			"aiven_kafka_acl":                    datasourceKafkaACL(),
 			"aiven_kafka_topic":                  datasourceKafkaTopic(),
+			"aiven_elasticsearch_acls":           datasourceElasticsearchACLs(),
```

Comment:
I think singular form would be better because everything else is in that form and ACL naturally contains multiple entries.

Reply:
indeed, I will change that, thanks